src/qt/locale/bitcoin_pl.ts

```diff
@@ -0,0 +1,2749 @@
+    </message>
+    <message>
+        <location line="+14"/>
+        <source>Copy the currently selected address to the system clipboard</source>
+        <translation>Skopiuj zaznaczony adres do schowka</translation>
+    </message>
+    <message>
+        <location line="-11"/>
+        <source>&amp;New Address</source>
+        <translation>&amp;Nowy adres</translation>
+    </message>
+    <message>
+        <location line="-46"/>
+        <source>These are your VERGE addresses for receiving payments. You may want to give a different one to each sender so you can keep track of who is paying you.</source>
+        <translation>To są  Twoje adresy VERGE do otrzymywania płatności. Możesz dać osobny adres każdemu odbiorcy aby śledzić kto Ci płaci.</translation>
+    </message>
+    <message>
+        <location line="+60"/>
+        <source>&amp;Copy Address</source>
+        <translation>&amp;Skopiuj adres</translation>
+    </message>
+    <message>
+        <location line="+11"/>
+        <source>Show &amp;QR Code</source>
+        <translation>Show &amp;Kod QR</translation>
+    </message>
+    <message>
+        <location line="+11"/>
+        <source>Sign a message to prove you own a VERGE address</source>
+        <translation>Podpisz wiadomość aby udowodnić, ze jesteś właścicielem adresu</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Sign &amp;Message</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+25"/>
+        <source>Delete the currently selected address from the list</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="-14"/>
+        <source>Verify a message to ensure it was signed with a specified VERGE address</source>
+        <translation>Weryfikuj wiadomość aby upewnić się, że została podpisana podanym adresem VERGE</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>&amp;Verify Message</source>
+        <translation>&amp;Weryfikuj wiadomość</translation>
+    </message>
+    <message>
+        <location line="+14"/>
+        <source>&amp;Delete</source>
+        <translation>&amp;Usuń</translation>
+    </message>
+    <message>
+        <location filename="../addressbookpage.cpp" line="+65"/>
+        <source>Copy &amp;Label</source>
+        <translation>Copy &amp;Etykieta</translation>
+    </message>
+    <message>
+        <location line="+2"/>
+        <source>&amp;Edit</source>
+        <translation>&amp;Edytuj</translation>
+    </message>
+    <message>
+        <location line="+250"/>
+        <source>Export Address Book Data</source>
+        <translation>Eksportuj dane książki adresowej</translation>
+    </message>
+    <message>
+        <location line="+1"/>
+        <source>Comma separated file (*.csv)</source>
+        <translation>Plik (*.csv)</translation>
```

Comment:
"Plik oddzielony przecinkami" instead of "Plik"

Reply:
Używa się tej nazwy ? Tak wydawało mi się dostatecznie jasne

Reply:
Comma separated jasno mówi że plik ma być oddzielony przecinkami. Czasami używa się średników czy pipe-ów, więc moim zdaniem dobrze byłoby jasno powiedzieć czego oczekujemy.

Reply:
Jasne, rozumiem :)